actions/helm-deploy/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Container image that runs your code
+FROM alpine/helm:3.2.1
+
+ARG KUBECTL_VERSION=1.18.12
+
+RUN apk add --update --no-cache curl bash git && \
+    curl -LO https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL_VERSION}/bin/linux/amd64/kubectl && \
+    mv kubectl /usr/bin/kubectl && \
+    chmod +x /usr/bin/kubectl
+
+COPY entrypoint.sh /entrypoint.sh
+
+ENTRYPOINT ["/entrypoint.sh"]
```

Comment:
You can reuse the Dockerfile that we use for locking namespaces, that way it will reuse all the layers: https://github.com/broadinstitute/datarepo-actions/blob/master/actions/lock-namespace/Dockerfile

Reply:
Hmm that doesn't install helm though

Reply:
I just changed this to use the same base image.